scapy/sendrecv.py

```diff
@@ -844,6 +845,11 @@ def sniff(count=0, store=True, offline=None, prn=None, lfilter=None,
             sniff_sockets[opened_socket] = "socket0"
     if offline is not None:
         flt = karg.get('filter')
+
+        if not TCPDUMP and not flt is None:
+            message = "tcpdump is not available. Cannot use filter!"
+            raise Scapy_Exception(message)
```

Comment:
Do you think we should try to fallback on `conf.use_pcap` when tcpdump isn’t available ?

Reply:
Also: you don't need tcpdump with `conf.pcap` set to `True`. That's why it's failing on windows, where it won't use tcpdump anyways